katportalclient/test/test_client.py

```diff
@@ -896,6 +896,7 @@ def test_sensor_history_multiple_sensors_valid_times(self):
 
         histories = yield self._portal_client.sensors_histories(
             sensor_name_filter, start_time_sec=0, end_time_sec=time.time())
+        print('histories == {}'.format(histories))
```

Comment:
Are these print statements just for debugging and will be removed once that is complete?

Reply:
yes, will remove it in the next commit.